smarter-suggestions-in-the-awesomebar.toml

```diff
@@ -1,5 +1,38 @@
-[metrics]
+[experiment]
+segments = ['urlbar_users', 'searchsuggestion_users']
+
+[segments.urlbar_users]
+select_expression = 'COALESCE(LOGICAL_OR((events.key like "urlbar#%#%") AND (events.value > 0)), FALSE)'
+data_source = "clients_daily_unnested"
+
+[segments.data_sources.clients_daily_unnested]
+from_expression = """(
+  SELECT 
+    submission_date,
+    client_id,
+    events
+  FROM
+    `moz-fx-data-shared-prod`.telemetry.clients_daily
+  CROSS JOIN
+    UNNEST(scalar_parent_telemetry_event_counts_sum) AS events
+)"""
+window_start = -7
+window_end = 0
+
+[segments.searchsuggestion_users]
+select_expression = """
+  COALESCE((SUM(COALESCE((
+    SELECT SUM(x.value) FROM UNNEST(scalar_parent_urlbar_picked_searchsuggestion_sum
+  ) x), 0)) > 0), FALSE)
+"""
+data_source = "clients_daily_custom_windows"
+
+[segments.data_sources.clients_daily_custom_windows]
+from_expression = "mozdata.telemetry.clients_daily"
+window_start = -7
+window_end = 0
 
+[metrics]
 weekly = [
 	"urlbar_search_count", 
```

Comment:
You don't have to list the outcome metrics here.
Once the outcome has been merged, you can select it in Experimenter and then all the metrics that are defined in the outcome config will be computed automatically.

Reply:
I see, by Experimenter, do you mean there will be things to choose or click on here: https://experimenter.services.mozilla.com/nimbus/firefox-suggest-history-vs-offline

Reply:
When creating a new experiment you can select Primary and Secondary outcomes in the Metrics tab:
![Screen Shot 2021-08-19 at 15 37 11](https://user-images.githubusercontent.com/1239705/130153252-7c35581c-3afc-4950-bc3a-5716835a9d1e.png)

Once this PR gets merged, "Firefox Suggest" will appear in that dropdown list. When selected and the experiment has been launched, the metrics defined in the outcome will automatically be computed.

Now that these experiments have already been launched, someone from the Experimenter team wll need to update the experiment definitions manually and add this outcome to it (maybe @jaredlockhart can do that?).

There is also some documentation around outcomes here that might be helpful: https://experimenter.info/jetstream/outcomes/

Reply:
Yep can do, generally we don't want to alter experiments after they've launched but we can make an exception in this case. @rebecca-burwei Just ping me with the links to the experiments you want to change and which outcomes you want to add in #experimenter-dev and I can add them 👍 Just keep in mind that generally altering an experiment after it launches should be avoided as much as possible.

Reply:
Thanks @scholtzan and @jaredlockhart ! Will do